csrec/DALFactory.py

```diff
@@ -0,0 +1,20 @@
+__author__ = "Angelo Leto"
+__email__ = "angleto@gmail.com"
+
+import MemDAL
+import MongoDAL
```

Comment:
this makes necessary pymongo even if we chose the 'mem' option. is it possible to import pymongo (or any other library to access the DB) only when the DB is requested?